windows.Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM lherman/mediadevices-base
+
+ENV TARGET_OS=windows \
+    PYTHONUNBUFFERED=1 \
+    # Somehow libopus.dll is installed under /usr/local/bin, so we need to add this
+    # path to WINEPATH so that wine can find this library
+    WINEPATH=/usr/local/bin \
+    # Go and C default environments
+    GOOS=windows \
+    GOARCH=amd64 \
+    CGO_ENABLED=1 \
+    CC=x86_64-w64-mingw32-gcc
```

Comment:
CXX will also be needed to use OpenH264 and windows camera driver.

Reply:
Ah, that's true. I forgot about that.